src/OneExplorer/OneExplorer.ts

```diff
@@ -360,26 +362,62 @@ input_path=${modelName}.${extName}
   }
 
   /**
-   * Search .cfg files in the same directory
+   * Search corresponding .cfg files inside the workspace
+   * for the given baseModelNode
+   *
+   * @param baseModelNode a Node of the base model
    */
-  private searchPairConfig(node: Node) {
-    console.assert(node.type === NodeType.baseModel);
+  private searchPairConfig(baseModelNode: Node) {
+    console.assert(baseModelNode.type === NodeType.baseModel);
+
+    /**
+     * Returns every file inside directory
+     * @todo Check soft link
+     * @param root
+     * @returns
+     */
+    const readdirSyncRecursive = (root: string): string[] => {
+      if (fs.statSync(root).isFile()) {
+        return [root];
+      }
 
-    const files = fs.readdirSync(node.parent);
+      let childs: string[] = [];
+      if (fs.statSync(root).isDirectory()) {
+        fs.readdirSync(root).forEach(val => {
+          childs = childs.concat(readdirSyncRecursive(path.join(root, val)));
+        });
+      }
+      return childs;
+    };
 
-    for (const fname of files) {
-      const fpath = path.join(node.parent, fname);
-      const fstat = fs.statSync(fpath);
+    // Get the list of all the cfg files inside workspace root
+    const confs =
+        readdirSyncRecursive(this.workspaceRoot!.fsPath).filter(val => val.endsWith('.cfg'));
 
-      if (fstat.isFile() && fname.endsWith('.cfg')) {
-        const parsedInputPath = this.parseInputPath(fpath, node.path);
-        if (parsedInputPath) {
-          const fullInputPath = path.join(node.parent, parsedInputPath);
-          if (this.comparePath(fullInputPath, node.path)) {
-            const pairNode = new Node(NodeType.config, [], vscode.Uri.file(fpath));
-            this.searchChildModels(pairNode);
-            node.childNodes.push(pairNode);
-          }
+    for (const conf of confs) {
+      const parsedObj = ConfigObj.parse(vscode.Uri.file(conf));
+      if (!parsedObj) {
+        Logger.info('OneExplorer', `Failed to open file ${conf}`);
+        return;
```

Comment:
What about using `continue`? It would be nice to proceed without the error file. And we can add extra mark like an exclamation point mark on the error file.

Reply:
@jyoungyun Oh it must be a bug. thanks for finding out.